lib/index.js

```diff
@@ -55,9 +55,9 @@ function Queue(name, opts, fn) {
   this.keys = {
     IN_PROGRESS: 'inProgress',
     QUEUE: 'queue',
-    ACK: 'ack',
     RECLAIM_START: 'reclaimStart',
-    RECLAIM_END: 'reclaimEnd'
+    RECLAIM_END: 'reclaimEnd',
+    ACK: 'ack'
   };
```

Comment:
From what I could tell, `@ndhoule/each` seemed to respect the order of the keys. 
`@ndhoule/each` is used by `storage` when migrating data from localstorage to in memory